HoloRepositoryUI/client/src/components/shared/App.tsx

```diff
@@ -69,7 +69,9 @@ class App extends Component<any, IAppState> {
       for (const pid in combinedResult) {
         const studies = combinedResult[pid];
         const patient = patients[pid];
-        patient.imagingStudies = studies;
+        if (patient !== undefined) {
```

Comment:
When the practitioner id is wrong, this code will crash because patient is undefined, but now we use the hard-coded practitioner list, it won't happens anymore